### ISSUE TITLE
Feature request: allow labels in model piping

### ISSUE THREAD

**Issue by billdenney** [nlmixr2/rxode2#351]
When adding a new parameter with model piping, the parameter label will be empty.  I think it would be helpful to also be able to add a `label()`.

Is there already a way to do that?

**Comment by mattfidler** [comment]
No, but I could see:

```r
f %>% ini(parameter="label")
```

Or 

```r
f %>% ini(parameter="label")
```

**Comment by billdenney** [comment]
As I'm working on some models where I'm making extensive use of piping, I had a new idea here.  How about the following interface:

```
f %>% ini(parameter=label("new label"))
```

That way, it will be clearer for a user what is happening and simpler for us to find when a label is being changed.  Thoughts?

**Comment by mattfidler** [comment]
That could work too. 

If we implement that approach, probably a error when trying `ini(parameter="new label")` telling that it should be `ini(parameter=label("new label"))`

**Comment by billdenney** [comment]
This is fixed in #385.
